benchexec/tools/owic.py

```diff
@@ -0,0 +1,56 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.template
+import benchexec.result as result
+
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    This class serves as tool adaptor for OWI (https://github.com/OCamlPro/owi)
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("owic", subdir="bin")
+
+    def version(self, executable):
+        return self._version_from_tool(executable, "--version")
+
+    def name(self):
+        return "owic"
```

Comment:
Is the tool name "owi" or "owic"? Here the proper tool name should be returned.

And is the spelling "owi", "OWI", or "Owi"? I can find all three of these spellings here and in your readme.

Reply:
The tool executable is named `owic`. But the display name should be "OWI" as specified in the benchmark and category definitions https://gitlab.com/sosy-lab/test-comp/bench-defs/-/merge_requests/114. Will update this accordingly, thanks!